src/process/posix/SDL_posixprocess.c

```diff
@@ -0,0 +1,371 @@
+/*
+  Simple DirectMedia Layer
+  Copyright (C) 1997-2024 Sam Lantinga <slouken@libsdl.org>
+
+  This software is provided 'as-is', without any express or implied
+  warranty.  In no event will the authors be held liable for any damages
+  arising from the use of this software.
+
+  Permission is granted to anyone to use this software for any purpose,
+  including commercial applications, and to alter it and redistribute it
+  freely, subject to the following restrictions:
+
+  1. The origin of this software must not be misrepresented; you must not
+     claim that you wrote the original software. If you use this software
+     in a product, an acknowledgment in the product documentation would be
+     appreciated but is not required.
+  2. Altered source versions must be plainly marked as such, and must not be
+     misrepresented as being the original software.
+  3. This notice may not be removed or altered from any source distribution.
+*/
+#include "SDL_internal.h"
+
+#ifdef SDL_PROCESS_POSIX
+
+#include <errno.h>
+#include <signal.h>
+#include <spawn.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <sys/wait.h>
+
+#include "../SDL_sysprocess.h"
+
+#define READ_END 0
+#define WRITE_END 1
+
+struct SDL_ProcessData {
+    pid_t pid;
+    int stdin_pipe[2];
+    int stdout_pipe[2];
+    int stderr_pipe[2];
```

Comment:
`SDL_ProcessData` does not need to save both pipe ends. This saves an amazing 12 bytes :)
The same applies to the Windows backend.

Reply:
Done!